server/dbs/postgres_sqlalchemy.py

```diff
@@ -84,6 +83,10 @@ def _isFunctionAllowed(self, funcname):
                 if (func[1] not in ('i', 's') and
                         self._allowedFunctions.get(func[0], False)):
                     self._allowedFunctions[func[0]] = False
+            # Whitelist some functions if they exist
+            for func in funcs:
+                if func[0] in ('pg_sleep', ):
+                    self._allowedFunctions[func[0]] = True
```

Comment:
Won't this make it so users are able to call this even in production? Is that something we are OK with allowing? My only worry is it could be used to just leave connections open to the database for indefinite periods.